mapswipe_workers/mapswipe_workers/utils/api_calls.py

```diff
@@ -27,33 +29,76 @@ def geojsonToFeatureCollection(geojson: dict) -> dict:
     return geojson
 
 
-def query_osm(changeset_id: int):
+osm_results = {}
+
+
+def chunks(_list, n_objects):
+    return [
+        _list[i * n_objects : (i + 1) * n_objects]
+        for i in range((len(_list) + n_objects - 1) // n_objects)
+    ]
+
+
+def query_osm(changeset_ids: list):
     """Get data from changesetId."""
-    url = OSM_API_LINK + f"changeset/{changeset_id}.json"
+    id_string = ""
+    for id in changeset_ids:
+        id_string += f"{id},"
+
+    id_string = id_string[:-1]
 
+    url = OSM_API_LINK + f"changesets?changesets={id_string}"
+    logger.info(url)
```

Comment:
Hey,
let's remove the logging of the url.
But it would be nice to add a log after the query and logging the progress. Something like `finished query 1/15`